Modules/Server/Sources/PocketCastsServer/Public/Sharing/Structs/ServerStructs.swift

```diff
@@ -276,11 +277,27 @@ public struct DiscoverItem: Decodable {
         case isSponsored = "sponsored"
         case sponsoredPodcasts = "sponsored_podcasts"
         case expandedTopItemLabel = "expanded_top_item_label"
-        case type, title, source, regions, curated, uuid, popular
+        case type, title, source, regions, curated, uuid, popular, id
+    }
+
+    public init(id: String? = nil, uuid: String? = nil, title: String? = nil, type: String? = nil, summaryStyle: String? = nil, expandedStyle: String? = nil, source: String? = nil, sponsoredPodcasts: [CarouselSponsoredPodcast]? = nil, expandedTopItemLabel: String? = nil, curated: Bool? = nil, regions: [String], isSponsored: Bool? = nil, popular: [Int]? = nil) {
+        self.id = id
+        self.uuid = uuid
+        self.title = title
+        self.type = type
+        self.summaryStyle = summaryStyle
+        self.expandedStyle = expandedStyle
+        self.source = source
+        self.sponsoredPodcasts = sponsoredPodcasts
+        self.expandedTopItemLabel = expandedTopItemLabel
+        self.curated = curated
+        self.regions = regions
+        self.isSponsored = isSponsored
+        self.popular = popular
```

Comment:
Can't we just do:

```suggestion
    public init(id: String? = nil, title: String? = nil, source: String? = nil, regions: [String]) {
        self.id = id
        self.title = title
        self.source = source
        self.regions = regions
```

Since it's what is really used by the code and the rest we can omit?

Reply:
Thanks! Good catch 👍